hbook/hbook.go

```diff
@@ -40,6 +40,17 @@ func (r Range) Width() float64 {
 	return math.Abs(r.Max - r.Min)
 }
 
+// Count is a 1-dim binned information, made of
+// a X range, a value and its uncertainties.
+type Count struct {
```

Comment:
could we also have an example for `Count` (with perhaps also modifying an example of `H1D` to use `Counts()`) ?

Reply:
I did two examples, one with just a `hbook.Count` type and the other with `hbook.H1D` (but I didn't modify existing H1D example). Let me know if you think more (or different) examples would be useful - I am happy to add them.